client/src/components/common/Button/index.css

```diff
@@ -7,12 +7,8 @@
     font-size: 20px;
     line-height: 23px;
     border-radius: 15px;
-    position: absolute;
-    width: 207px;
-    height: 33px;
-    left: 84px;
-    top: 344px;
     display: flex;
     align-items: center;
     text-align: center;
+    padding: 10px 20px;
```

Comment:
remove it and make class for your button component and add this line

Reply:
and add in your case `pointer-events: none;`
because of your button in the link tag